doc/examples/segmentation/plot_random_walker_segmentation.py

```diff
@@ -17,22 +17,27 @@
 values, and use the random walker for the segmentation.
 
 .. [1] *Random walks for image segmentation*, Leo Grady, IEEE Trans. Pattern
-       Anal. Mach. Intell. 2006 Nov; 28(11):1768-83
+       Anal. Mach. Intell. 2006 Nov; 28(11):1768-83 DOI:10.1109/TPAMI.2006.233
 
 """
 import numpy as np
 import matplotlib.pyplot as plt
 
 from skimage.segmentation import random_walker
 from skimage.data import binary_blobs
+from skimage.exposure import rescale_intensity
 import skimage
 
 # Generate noisy synthetic data
 data = skimage.img_as_float(binary_blobs(length=128, seed=1))
 data += 0.35 * np.random.randn(*data.shape)
+data = rescale_intensity(data, in_range=(-0.35, 1.35))
+
+# The range of the binary image spans over (-1, 1).
+# We choose the markers as the hotest and coldest pixels.
```

Comment:
Maybe "We choose the hotest and the coldest pixels as markers." would be better?